gems/pending/spec/spec_helper.rb

```diff
@@ -12,6 +14,8 @@
 # Initialize the global logger that might be expected
 require 'logger'
 $log ||= Logger.new("/dev/null")
+# $log ||= Logger.new(STDOUT)
+# $log.level = Logger::DEBUG
```

Comment:
This seem unrelated to this PR.  Can we make a debugging environmental variable that turns on these switches in a followup PR?